### ISSUE TITLE
Use validates_url inside a AR validate method

### ISSUE THREAD

**Issue by JulienItard** [perfectline/validates_url#78]
Hello !

I'd like to call `validates_url` inside a method. Is it possible to do it ?

With this test code, it raises undefined method.

```
validate :my_validation

def my_validation 
  validates_url(:url) && other validation && other validation
end
```

Thanks a lot

**Comment by kritik** [comment]
Yes, if you will initialize validator yourself. But you don't want to do that in 99%. What is your case?

**Comment by JulienItard** [comment]
Thanks for your quick anwser.

I have field that can be an URL, a `mailto:...` or a `tel:...`. 

I'd like to create a custom method and use your validates_url method when it's an URL.
Something like : 

```
def my_validation
  validates_url(:url) || url.start_with?("mailto:") || url.start_with?("tel:")
end
```
